profiling/src/profiling/mod.rs

```diff
@@ -905,6 +908,42 @@ impl Profiler {
                 sample_values.extend_from_slice(&values[3..5]);
             }
 
+            let fiber = unsafe { ddog_php_prof_get_active_fiber() };
+            if !fiber.is_null() {
+                unsafe {
+                    let functionname = Some(String::from_utf8_lossy(
+                        ddog_php_prof_zend_string_view(
+                            (*(*fiber).fci_cache.function_handler)
+                                .op_array
```

Comment:
This needs a check whether it's actually an op_array... We have a helper function `extract_function_name()` in stack_walking.rs. He Handling of fci_cache.object then probably is unnecessary too?

Reply:
Thanks for pointing that out, I hated this part 😉